components/class-m-chart-highcharts.php

```diff
@@ -620,12 +620,22 @@ public function fix_null_values( $value ) {
 	 * @return array an array of themes
 	 */
 	public function get_themes() {
+		$cache_key = 'm-chart-highcharts-available-themes';
+
+		if ( $themes = wp_cache_get( $cache_key ) ) {
```

Comment:
Take a look here for how I'd prefer the caching key stuff handled (i.e. use the group parameter for the name spacing):

https://github.com/methnen/m-chart-highcharts-library/blob/06b0177d2fb1fa5ceab70210be77e3c9a33e4d70/components/class-m-chart-highcharts.php#L126